engine/apps/base/models/live_setting.py

```diff
@@ -173,10 +187,21 @@ def get_setting(cls, setting_name):
     def populate_settings_if_needed(cls):
         settings_in_db = cls.objects.filter(name__in=cls.AVAILABLE_NAMES).values_list("name", flat=True)
         setting_names_to_populate = set(cls.AVAILABLE_NAMES) - set(settings_in_db)
+        if len(setting_names_to_populate) == 0:
+            return
 
         for setting_name in setting_names_to_populate:
             cls.objects.create(name=setting_name, value=cls._get_setting_from_setting_file(setting_name))
 
+        cls.revalidate_settings()
```

Comment:
Since I missed the edit case in the UI this section is only to cover initial population and when a setting is deleted and repopulated.  This needs to not happen most of the time since this function is called on every get.
The update case is now handled by calling validate_settings in perform_update